ffi/diplomat/c/examples/fixeddecimal_tiny/Makefile

```diff
@@ -15,54 +15,51 @@ GCC := gcc
 CLANG := clang-14
 LLD := lld-14
 
+baked_data/macros.rs:
+	cargo run -p icu_datagen -- --locales en bn --keys all --fallback expand --format mod --use-separate-crates --out baked_data
```

Comment:
Observation: Previously you had `--keys "decimal/symbols@1"` and now you have `--keys all`. This seems to slow down CI: running this step takes about 1m 18s on CI, whereas previously it took 1s.

You already depend on `icu_capi` with all features disabled except for `icu_decimal`. Can you generate the data with only the keys for the decimal component?

However, overall CI speed does not seem to be affected, so this is fine for now if it's not an easy fix.

Reply:
Yes, this actually made me notice that we should probably generate empty implementations for all keys, otherwise compiled data will not compile. I'll make that change in datagen and then undo the `--keys all`

Reply:
In a separate PR